packages/ir-sdk/fern/apis/ir-types-latest/definition/http.yml

```diff
@@ -201,6 +201,13 @@ types:
       error: errors.DeclaredErrorName
 
   Pagination:
+    docs: |
+      If set, the endpoint will be generated with auto-pagination features.
+    union:
+      cursor: CursorPagination
+      offset: OffsetPagination
+
+  CursorPagination:
```

Comment:
this will be a breaking change so we'll have to do an IR migration (assume you're already tracking)

Reply:
I figured we would be able to get away with it because there are zero consumers of it (both in the `fern` compiler as well as all the other generators). As long as we update the respective IR SDKs this should be ok, no? I might be missing something though.